docker/Dockerfile

```diff
@@ -27,6 +27,9 @@ COPY docker/girder.local.conf /etc/girder.cfg
 # Copy over mongochemserver
 COPY . /mongochemserver
 
+# Install girder jobs plugin (needed by mongochemserver)
+RUN pip install -e /girder/plugins/jobs
```

Comment:
I modified the setup.py file to do so (and removed this line in the Dockerfile). However, it looks like it installs `girder-jobs` from the web. This might potentially be different from the one in `/girder/plugins`.